notebooks/inference_SavedModel_VertexEndpoint.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<blockquote><strong>NOTE</strong>: The semantic segmentation prediction doesn't look good on this notebook. This is because the purpose of this notebook is not about showing the result from the nicely fine-tuned model but demonstrating how to perform inferences of the model from the Vertex Pipeline.</blockquote><p> </p><p>Our Space application's predictions are good. Can't we fetch that model from the artifact store? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>